statistics/statistics_test.go

```diff
@@ -116,16 +221,15 @@ func TestStatisticsProcessorMetrics(t *testing.T) {
 				metrics[i] = plugin.MetricType{
 					Data_:      data,
 					Namespace_: core.NewNamespace("foo", "bar"),
-					Timestamp_: time.Now(),
+					Timestamp_: time[i],
 				}
 			}
 			var buf bytes.Buffer
 			enc := gob.NewEncoder(&buf)
 			enc.Encode(metrics)
 
-			movingAverageObj := New()
-
-			_, receivedData, _ := movingAverageObj.Process("snap.gob", buf.Bytes(), nil)
+			statisticObj := New()
+			_, receivedData, _ := statisticObj.Process("snap.gob", buf.Bytes(), config)
```

Comment:
This test to ensure that if a value of unknown type was passed in that the exception would be handled properly.